exercicios-yuri/exercicio-api/main.go

```diff
@@ -0,0 +1,250 @@
+package main
+
+import (
+	"errors"
+	"log"
+	"net/http"
+	"strconv"
+	"time"
+
+	"github.com/gin-gonic/gin"
+)
+
+func main() {
+
+	u := newItemUsecase()
+	h := newHandler(u)
+
+	router := gin.Default()
+
+	router.GET("/", h.home)
+	router.GET("/hello", h.hello)
+	router.POST("/bye", h.bye)
+
+	router.POST("/items", h.saveItem)
+	router.GET("/items", h.listItems)
+
+	router.GET("/items/:id", h.itemById)
+	router.PUT("/items", h.changeItem)
+	router.DELETE("/items/:id", h.deleteItem)
+
+	log.Println("Server started at http://localhost:8080/")
+
+	if err := router.Run(":8080"); err != nil {
+		log.Fatal(err)
+	}
+}
+
+// /////////////////////////////////////////////////////////////////////////////
+// Global error
+// /////////////////////////////////////////////////////////////////////////////
+var errNotFound = errors.New("not found")
```

Comment:
Si lo vas a usar como constante, definirla como constante

```suggestion
const ERR_NOT_FOUND = errors.New("not found")
```